src/platforms/react-native/manual-setup/hermes.mdx

```diff
@@ -32,6 +32,14 @@ If you are manually bundling and building React Native apps, follow this three-s
 
 ### Compile Sourcemaps
 
+<Note>
+
+Currently, there is an issue when bundling the `hermes` bundle and `source maps` manually.
+We recommend not bundling these manually until this is fixed.
+Follow the [issue](https://github.com/facebook/react-native/issues/34043) for more details.
```

Comment:
@imatwawana Do you know if linking issues work within a `Note`? Using the [ ] ( ) format, it's not working in the preview.

Reply:
Adding: as long as you leave a line break between the text of the note and the opening and closing Note tags, markdown formatting should always work.

Reply:
Oh that was the typo, thanks.